Engine/DataFeeds/TimeSlice.cs

```diff
@@ -293,7 +312,7 @@ private static bool HandleOptionData(DateTime algorithmTime, BaseData baseData,
             var symbol = baseData.Symbol;
             
             OptionChain chain;
-            var canonical = Symbol.Create(symbol.ID.Symbol, SecurityType.Option, symbol.ID.Market);
+            var canonical = Symbol.CreateOption(symbol.Underlying, symbol.ID.Market, default(OptionStyle), default(OptionRight), 0, SecurityIdentifier.DefaultDate);
```

Comment:
Why can't we use `Symbol.Create` directly anymore here? Requiring consumers to know how to define a canonical symbol isn't the responsibility of the consumer. Perhaps `Symbol` needs a method, such as `Symbol.CreateCanonicalOption` to hide the `default(T)` values being specified which are, arguably, internal concerns of `SecurityIdentifier` and the `Symbol` implementations